src/elm/Pages/Build/View.elm

```diff
@@ -615,32 +613,24 @@ viewServiceLogs msgs shift rm service =
 -}
 viewLogLines : LogsMsgs msg -> FollowResource msg -> Org -> Repo -> BuildNumber -> String -> ResourceID -> LogFocus -> Maybe (WebData Log) -> Int -> Bool -> Html msg
 viewLogLines msgs followMsg org repo buildNumber resource resourceID logFocus maybeLog following shiftDown =
-    let
-        decodedLog =
-            toString maybeLog
-
-        fileName =
-            downloadFileName org repo buildNumber resource resourceID
-    in
     div
         [ class "logs"
         , Util.testAttribute <| "logs-" ++ resourceID
         ]
     <|
         case Maybe.withDefault RemoteData.NotAsked maybeLog of
-            RemoteData.Success _ ->
-                if logEmpty decodedLog then
```

Comment:
we no longer need this code because a decodedLog will never be empty.
if the log were empty, it would be decoded with the helpful message and be safe to render.